assets/js/modules/analytics/components/settings/SettingsForm.js

```diff
@@ -36,10 +36,11 @@ import { MODULES_ANALYTICS } from '../../datastore/constants';
 import { MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
 import SettingsControls from './SettingsControls';
 import GA4SettingsControls from './GA4SettingsControls';
+import EntityOwnershipChangeNotice from '../../../../components/settings/EntityOwnershipChangeNotice';
 import { isValidAccountID } from '../../util';
 const { useSelect } = Data;
 
-export default function SettingsForm() {
+export default function SettingsForm( { hasModuleAccess = true } ) {
```

Comment:
Given we need to check GA4 access separately, it might make more sense to only pass this as a prop to `*SettingsControls` and below and use selectors in this component and above.